### ISSUE TITLE
Question about results on FUSEG dataset

### ISSUE THREAD

**Issue by mlyg** [uwm-bigdata/wound-segmentation#5]
Firstly thank you very much for creating this dataset and challenge.

Looking at the results from your paper 'Fully automatic wound segmentation with deep convolutional neural networks', I was very impressed by the high DSC scores of >0.90. I was wondering whether those DSC scores are based on evaluation on the original images, or on the cropped image patches generated by YOLOv3?

**Comment by Pele324** [comment]
> Firstly thank you very much for creating this dataset and challenge.
> 
> Looking at the results from your paper 'Fully automatic wound segmentation with deep convolutional neural networks', I was very impressed by the high DSC scores of >0.90. I was wondering whether those DSC scores are based on evaluation on the original images, or on the cropped image patches generated by YOLOv3?

Hi @mlyg, the scores were evaluated on the original images.

**Comment by mlyg** [comment]
Thanks for the answer. 

Can I also just check that the dataset used in the paper is the same one as for the FUSeg2021 challenge? In other words, should we also be expecting DSC scores in excess of 0.90 on this challenge dataset?

**Comment by Pele324** [comment]
> Thanks for the answer.
> 
> Can I also just check that the dataset used in the paper is the same one as for the FUSeg2021 challenge? In other words, should we also be expecting DSC scores in excess of 0.90 on this challenge dataset?

From the observations that wounds can be very small in the images, we believed that cropped images are better for training our deep learning models. Thus the dataset used in the paper consists of cropped images. We would like to give challenge participants the freedom of choosing their own pre-processing methods. So we provide full images in the challenge dataset. In the paper we tested the vanilla U-Net is able to achieve 0.90 DSC. With smarter ideas and networks, I believe 0.9 is just the baseline of this problem.

**Comment by shahzad-ali** [comment]
> From the observations that wounds can be very small in the images, we believed that cropped images are better for training our deep learning models. Thus the dataset used in the paper consists of cropped images. We would like to give challenge participants the freedom of choosing their own pre-processing methods. So we provide full images in the challenge dataset. In the paper we tested the vanilla U-Net is able to achieve 0.90 DSC. With smarter ideas and networks, I believe 0.9 is just the baseline of this problem.

@Pele324: Thanks for sharing your viewpoint. I also kept believing that getting a dice score >0.9 would be a great start and put a lot of efforts achieving this goal but this didn't happen 😭

Surprisingly, the **[Foot Ulcer Segmentation Challenge Leaderboard](https://uwm-bigdata.github.io/wound-segmentation/)** revealed the best dice score as 0.8880 by the team securing the top position. **Would you please comment on this? If possible, please share the possible reasons behind the failure of the top model "U-Net+LinkNet" in outperforming a vanilla U-Net.**

What are the chances that the GT labels for the test fold will be released?

**Comment by uwm-bigdata** [comment]
> @Pele324: Thanks for sharing your viewpoint. I also kept believing that getting a dice score >0.9 would be a great start and put a lot of efforts achieving this goal but this didn't happen 😭
> 
> Surprisingly, the **[Foot Ulcer Segmentation Challenge Leaderboard](https://uwm-bigdata.github.io/wound-segmentation/)** revealed the best dice score as 0.8880 by the team securing the top position. **Would you please comment on this? If possible, please share the possible reasons behind the failure of the top model "U-Net+LinkNet" in outperforming a vanilla U-Net.**  

We tested U-Net on the challenge dataset and the dice was around 82%. You may already notice that the images we used in the paper are image patches cropped from the images of the challenge dataset (based on bounding boxes detection). In the paper we proposed this "detection->cropping->segmentation" pipeline. If we use "U-Net+LinkNet" as the segmentation module in this pipeline I think the dice score can be improved.
> 
> What are the chances that the GT labels for the test fold will be released?

We plan to keep FUSeg an open challenge so we won't release the GT for now. :sweat_smile:
